web/web.go

```diff
@@ -392,6 +392,11 @@ func New(logger log.Logger, o *Options) *Handler {
 
 	serveReactApp := func(w http.ResponseWriter, r *http.Request) {
 		f, err := ui.Assets.Open("/static/react/index.html")
+		defer func() {
+			if err := f.Close(); err != nil {
+				fmt.Fprintf(w, "Error closing index.html: %v", err)
```

Comment:
At this stage, HTML has been served and HTTP code has been written. I would just ignore the error:

defer func(){ f.Close() }

Also, this should be called after we have done the err check below.

Reply:
The linter complains if the error is unchecked. Do I ignore the error or the linter?

Reply:
Also, if ignoring the error, I could just use `defer f.Close()`,  right?